src/widget/OktaSignIn.js

```diff
@@ -36,18 +36,19 @@ var OktaSignIn = (function () {
         Router = require('v2/WidgetRouter');
       }
       if (widgetOptions.stateToken) {
+        router = bootstrapRouter.call(this, Router, authClient, widgetOptions, renderOptions, successFn, errorFn);
+        if (!widgetOptions.apiVersion) {
+          widgetOptions.apiVersion = router.settings.get('apiVersion');
```

Comment:
doesn't change `widgetOptions` that user passed in place?
if yes, we'd better create a new object.